libtransmission/file-capacity.cc

```diff
@@ -27,6 +27,11 @@
 #else
 #include <sys/quota.h> /* quotactl() */
 #endif
+
+#ifndef btodb
+#define btodb(num) ((num) >> 10)
+#endif
```

Comment:
The less magic numbers there are the better. Since NDK has `QIF_DQBLKSIZE_BITS`, I'd rather it be
```suggestion
#if !defined(btodb) && defined(QIF_DQBLKSIZE_BITS)
#define btodb(num) ((num) >> QIF_DQBLKSIZE_BITS)
#endif
```